test/parallel/test-debugger-object-type-remote-object.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const common = require('../common');
+
+common.skipIfInspectorDisabled();
+
+const fixtures = require('../common/fixtures');
+const startCLI = require('../common/debugger');
+
+const assert = require('assert');
+
+const cli = startCLI([fixtures.path('debugger/empty.js')]);
+
+function onFatal(error) {
+  cli.quit();
+  throw error;
+}
+
+cli.waitForInitialBreak()
```

Comment:
oy... rather than a long chain of thens... can we convert this into an async function with awaits please?

Reply:
@jasnell done 3f21ce57a3b5f9047203b1a581a91309ae27e4f5